.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:
 
+> **Unless you are cherry-picking an existing commit into a current release, ensure your pull request is targeting the `master` React Native branch.**
```

Comment:
maybe add "Unless you are [a React Native release maintainer and] cherry-picking..." or some other wording so it's clear that nearly all contributors never need to cherry pick.